src/library/crate/cratestorage.cpp

```diff
@@ -441,6 +457,26 @@ CrateTrackSelectResult CrateStorage::selectTrackCratesSorted(TrackId trackId) co
 }
 
 
+CrateTrackSelectResult CrateStorage::selectTracksSortedByCrateNameLike(const QString& crateName) const {
+    FieldEscaper escaper(m_database);
```

Comment:
I tried it the last time you mentioned it, and it didn't work. The name needs to be wrapped in kSqlLikeMatchAll (%). Also it needs to be inside quotes. bindValue() does not seem to do that. I can use it like this:
`bindValue("\'" + kSqlLikeMatchAll + crateNameLike + kSqlLikeMatchAll + "\'")`